Assets/GLTF/Scripts/GLTFLoader.cs

```diff
@@ -154,16 +155,28 @@ protected virtual GameObject CreateScene(Scene scene)
 
 			foreach (var node in scene.Nodes)
 			{
-				var nodeObj = CreateNode(node.Value);
+				var nodeObj = CreateNode(node);
 				nodeObj.transform.SetParent(sceneObj.transform, false);
 			}
 
+			foreach (var node in scene.Nodes)
+			{
+				CreateNodePrimitives(node.Value, _nodeMap[node.Id]);
+			}
+
+			foreach (var animation in _root.Animations)
+			{
+				AddAnimationToScene(scene, sceneObj, animation);
+			}
+
```

Comment:
Small nitpick, but you can get rid of some brackets to save some lines

Reply:
@bjornbytes likes the brackets, you and I dislike them :). I'd appreciate it if we resolved this with a street brawl!